addons/mail/static/src/new/discuss/channel_member_list.scss

```diff
@@ -0,0 +1,12 @@
+.o-mail-channel-member:hover {
+    background-color: map-get($grays, '300') !important;
+}
+
+.o-mail-channel-member-avatar-container {
+    width: 32px;
+    height: 32px;
+}
+
+.o-mail-channel-member-name {
+    min-width: 0;
+}
```

Comment:
I don't see a use for this rule. Is it still relevant?

Reply:
It was in the old code and I thought It is related to small device and I kept it. It doesn't have any effect now. I can delete it.

Reply:
Ok, will delete it in follow-up PR, as I don't see a use-case. Thanks!